src/cascadia/TerminalApp/TerminalApp.vcxproj

```diff
@@ -151,19 +96,14 @@
       <AdditionalIncludeDirectories>$(OpenConsoleDir)\dep\jsoncpp\json;%(AdditionalIncludeDirectories);</AdditionalIncludeDirectories>
     </ClCompile>
     <Link>
+      <!-- Manually link with the TerminalAppLib.lib we've built. -->
+      <AdditionalLibraryDirectories>$(_BinRoot)\TerminalAppLib;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>

```

Comment:
i thought `ProjectReference`ing a static lib did this automatically.

Reply:
Hmm. That one might have been left behind. There was a time in the life of this PR it was important, but no longer it seems.